scripts/complete_liquidity_provision.js

```diff
@@ -122,13 +122,17 @@ module.exports = async (callback) => {
       console.log("==> Skipping safe deployment and using brackets safeOwners")
       bracketAddresses = argv.brackets
       // Ensure that safes are all owned solely by masterSafe
-      await Promise.all(
+      const masterNotOnlyOwner = await Promise.all(
         bracketAddresses.map(async (safeAddr) => {
-          if (!(await isOnlySafeOwner(masterSafe.address, safeAddr))) {
-            callback(`Error: Bracket ${safeAddr} is not owned (or at least not solely) by master safe ${masterSafe.address}`)
-          }
+          !(await isOnlySafeOwner(masterSafe.address, safeAddr))
         })
       )
+      const badSafes = masterNotOnlyOwner.filter((_, i) => masterNotOnlyOwner[i] == true)
```

Comment:
diminutive warning: in js I would never use ==, but always ===. but here you can even leave it out.

Reply:
Is there a reason to leave it out here specifically?

Reply:
With leaving out, I mean:
```
const badSafes = masterNotOnlyOwner.filter((_, i) => masterNotOnlyOwner[i])
```
But I am braindead right now...

Reply:
```suggestion
      const badSafes = masterNotOnlyOwner.filter((_, i) => masterNotOnlyOwner[i])
```